deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/kustomization.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../../base/storetheindex-single
+  - ingress.yaml
+  - dido-snapshot.yaml
+
+namePrefix: ago-
+
+commonLabels:
+  name: ago
+
+secretGenerator:
+  - name: identity
+    behavior: replace
+    files:
+      - identity.key=identity.key.encrypted # 12D3KooWCSqbkKKKYW8RkqwRTt2M6xihv4LgmX5U1qDe3va7FR9Q
+
+configMapGenerator:
+  - name: config
+    behavior: replace
+    files:
+      - config=config.json
+
+patchesStrategicMerge:
+  - pvc_data.yaml
+  - pvc_addata.yaml
```

Comment:
This is not a patch since it's a brand new resource we are creating in this overlay, and therefore, should go under `resources`.